problem_builder/locale/config.yaml

```diff
@@ -9,6 +9,7 @@ locales:
     - fr  # French 
     - de_DE  # German
     - ja_JP  # Japanese
+    - pl
```

Comment:
It seems like the proper place for the translation files is probleim_builder/translations.